### ISSUE TITLE
Bug: Error Downloading File to a Set Folder

### ISSUE THREAD

**Issue by JPowell23** [iterate-ch/cyberduck#14127]
As of 2 updates ago and upon a fresh open of the application, I am no longer able to download a file to a folder (screenshot below).  The folder the app is trying to write to has read & write permissions for my user. 

**Current workaround:** 
1. Go to Cyberduck > Settings > Transfers > General > Download Folder. 
2. Open that dropdown menu and click "choose..." > Open. This re-sets that same folder. 
3. Retry downloading the file and should now complete. 

![CleanShot 2023-01-24 at 08 42 33@2x](https://user-images.githubusercontent.com/35232817/214342913-09af5fd5-d85e-41a4-acd2-6b620012d4ab.png)

**Comment by AliveDevil** [comment]
App Store? Might be a Sandbox issue on our side, not retrieving or storing the bookmark to get access to that folder later.

**Comment by JPowell23** [comment]
Right, this is the App Store version (8.5.3 [39083]) of the app. Not sure on the storing part. Looks like file path is saved as the folder name can be seen in the settings. Just have to set that path again within the app for it to know it can continue use it without error.

**Comment by AliveDevil** [comment]
@dkocher missing bookmark on app store for "Download File to" in preferences, and retrieving that bookmark again for downloads.
